src/main/webapp/metadataFragment.xhtml

```diff
@@ -130,7 +130,7 @@
                                                     <ui:repeat value="#{compoundValue.displayValueMap.entrySet().toArray()}" var="cvPart" varStatus="partStatus">
                                                         <c:set var="cvocOnCvPart" value="#{cvocConf.containsKey(cvPart.key.datasetFieldType.id)}"/>
                                                         <h:outputText value="#{dsf.datasetFieldType.displayFormat} " rendered="${!partStatus.first and !settingsWrapper.isCvocField(cvPart.key.datasetFieldType.id)}"/>
-                                                        <ui:fragment rendered="#{!cvocOnDsfApplies or !settingsWrapper.isCvocField(cvPart.key.datasetFieldType.id)}">
+                                                        <ui:fragment rendered="#{!(cvocOnDsfApplies or settingsWrapper.isCvocField(cvPart.key.datasetFieldType.id))}">
```

Comment:
In testing further, I think this has to be  `<ui:fragment rendered="#{!cvocOnCvPart and !cvocOnDsfApplies or !settingsWrapper.isCvocField(cvPart.key.datasetFieldType.id)}">` instead. When the ORCID and ROR scripts are configured on the author field and a plain text name/affilation are entered, the change currently in the PR suppresses the name as well as the unwanted copy of the affiliation. 

If I try to say the logic in English - this fragment is supposed to show when no script will be changing it, which occurs when 
- the child field itself is not a term-uri field (`cvocOnCvPart == false`), and
either
- there isn't a script configured on the parent field and there is a value for that script's term-uri field (`!cvocOnDsfApplies`) or
- the child field itself is not listed as a managed field (`!settingsWrapper.isCvocField(cvPart.key.datasetFieldType.id)` - isCvocField is technically true if it's a term-uri or managed field, but it can't be true from the field being a term-uri when cvocOnCvPart is false)

The original code failed in cases where cvocOnDsfApplies is false (primitive field, compound field with a script but no entry for the term-uri field) because it then displayed child fields that were term-uri fields (incorrect).

The PR as is fails when cvocOnDsfApplies is false (the author field with ORCID/ROR scripts, when there is no author identifier value) because in that case the author name is still a managed field and isn't rendered when it should be).

@ffritze - can you take a look and see if you agree with the logic or at least can confirm that my suggested change doesn't break your use case and also handles the case where the orcid and ror scripts are applied to the author field and there's an entry with no author id (i.e. user typed in a name that has no ORCID)? If so, can you change the PR and we'll continue through QA?